pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should model the POM like https://github.com/apache/flink-connector-elasticsearch/blob/main/pom.xml